pkg/commonconfig/commonconfig.go

```diff
@@ -18,19 +19,23 @@ const (
 )
 
 type CommonConfig struct {
-	configFlags genericclioptions.RESTClientGetter
+	ConfigFlags *genericclioptions.RESTClientGetter
```

Comment:
This is changed to a pointer because we can only initialize it at runtime when the flags are parsed